src/credentials/loginManager.ts

```diff
@@ -67,8 +68,7 @@ export class LoginManager {
             telemetryResult = 'Succeeded'
             return true
         } catch (err) {
-            // TODO: don't hardcode logic using error message, have a 'type' field instead
-            if (!(err as Error).message.includes('cancel')) {
+            if (TimeoutError.isCancelled(err)) {
```

Comment:
```suggestion
            if (!TimeoutError.isCancelled(err)) {
```
If the user cancelled then we don't want to show any message basically.